### ISSUE TITLE
Error Creating Service Account

### ISSUE THREAD

**Issue by kalahari** [kuberhost/kube-backup#14]
I get an error when applying 1_service_account.yaml on k8s 1.24
```
$ kubectl apply -f 1_service_account.yaml 
serviceaccount/kube-backup-user created
clusterrole.rbac.authorization.k8s.io/kube-backup-view-all created
rolebinding.rbac.authorization.k8s.io/psp:unprivileged created
error: resource mapping not found for name: "kube-backup-user" namespace: "" from "1_service_account.yaml": no matches for kind "ClusterRoleBinding" in version "rbac.authorization.k8s.io/v1beta1"
ensure CRDs are installed first
```

**Comment by kalahari** [comment]
Changing `rbac.authorization.k8s.io/v1beta1` to `rbac.authorization.k8s.io/v1` resolved the issue. I will submit a PR.

**Comment by kalahari** [comment]
fixed in PR #15
